### ISSUE TITLE
Notebooks don't connect with Google Colab's GPU

### ISSUE THREAD

**Issue by esgomezm** [vanvalenlab/deepcell-tf#383]
**Is your feature request related to a problem? Please describe.**
Whenever I run the notebooks using Google Colab, in the training I get the following message: 
```
X_train shape: (90, 216, 256, 1)
y_train shape: (90, 216, 256, 1)
X_test shape: (10, 216, 256, 1)
y_test shape: (10, 216, 256, 1)
Output Shape: (None, 216, 256, 2)
Number of Classes: 2
Training on 0 GPUs
```

and Google Colabs sends a warning message about being on a GPU environment without using any GPU. 

**Describe the solution you'd like**
It would be nice if the code connects with their GPUs/TPUs.

**Comment by willgraf** [comment]
Following up on this, I think it's likely that `tensorflow` was installed rather than `tensorflow-gpu`. The two packages have been merged with TensorFlow 2, and this issue should be resolved.

The next release of `deepcell` (based on TensorFlow 2.3.1) is now a pip package to make Colab installation easy.

**Comment by willgraf** [comment]
This is now resolved in release 0.8.1.
